models/event.go

```diff
@@ -12,18 +12,29 @@ type ContentPublished struct {
 
 // SearchDataImport provides event data for a search data import
 type SearchDataImport struct {
-	UID             string   `avro:"uid"`
-	URI             string   `avro:"uri"`
-	DataType        string   `avro:"data_type"`
-	JobID           string   `avro:"job_id"`
-	SearchIndex     string   `avro:"search_index"`
-	CDID            string   `avro:"cdid"`
-	DatasetID       string   `avro:"dataset_id"`
-	Keywords        []string `avro:"keywords"`
-	MetaDescription string   `avro:"meta_description"`
-	ReleaseDate     string   `avro:"release_date"`
-	Summary         string   `avro:"summary"`
-	Title           string   `avro:"title"`
-	Topics          []string `avro:"topics"`
-	TraceID         string   `avro:"trace_id"`
+	UID             string               `avro:"uid"`
+	URI             string               `avro:"uri"`
+	DataType        string               `avro:"data_type"`
+	JobID           string               `avro:"job_id"`
+	SearchIndex     string               `avro:"search_index"`
+	CDID            string               `avro:"cdid"`
+	DatasetID       string               `avro:"dataset_id"`
+	Keywords        []string             `avro:"keywords"`
+	MetaDescription string               `avro:"meta_description"`
+	ReleaseDate     string               `avro:"release_date"`
+	Summary         string               `avro:"summary"`
+	Title           string               `avro:"title"`
+	Topics          []string             `avro:"topics"`
+	TraceID         string               `avro:"trace_id"`
+	DateChanges     []ReleaseDateDetails `avro:"date_changes"`
+	Cancelled       bool                 `avro:"cancelled"`
+	Finalised       bool                 `avro:"finalised"`
+	ProvisionalDate string               `avro:"provisional_date"`
+	Published       bool                 `avro:"published"`
+}
+
+// ReleaseDateChange represent a date change of a release
```

Comment:
This is the same godoc error as occurred in the search data importer. But as this is a release it is just a suggestion to change it. Otherwise it can wait for a later fix:
```suggestion
// ReleaseDateDetails represent a date change of a release
```